# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/attr/hotfix-musl-basename.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- attr-2.5.2/tools/attr.c.vanilla	2025-01-01 02:55:23.324685038 +0100
+++ attr-2.5.2/tools/attr.c	2025-01-01 02:55:48.918018496 +0100
@@ -27,6 +27,7 @@
 #include <unistd.h>
 #include <errno.h>
 #include <string.h>
+#include <libgen.h>
 #include <locale.h>
 
 #include <attr/attributes.h>
